rmgpy/kinetics/diffusionLimited.py

```diff
@@ -152,29 +152,27 @@ def get_diffusion_limit(self, T, reaction, forward=True):
 
         return k_diff
 
-    def get_henry_law_constant_data(self, spec):
+    def get_henry_law_constant_data(self, spec, Ts=[]):
```

Comment:
Can you squash these fixes in with the other commits?

Reply:
Yes